assets/stylesheets/global/_base.scss

```diff
@@ -17,13 +17,17 @@ html._theme-dark {
   color-scheme: dark only;
 }
 
+:root {
+  --selectedFont: Arial;
+}
+
 body {
   height: 100%;
   margin: 0;
   overflow: auto;
   font-size: 1em;
   font-weight: normal;
-  font-family: $baseFont;
+  font-family: var(--selectedFont, $baseFont);
```

Comment:
In _variables.scss, we already define the variables `--baseFont` and `--monoFont` – I think we should re-use them instead of introducing another one.